walker_lake/walker_lake.ipynb

```diff
@@ -148,7 +152,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "crs=ccrs.UTM(11)"
+    "crs = ccrs.epsg(landsat_5_img.rio.crs.to_epsg())"
```

Comment:
This looks better than the hardcoded `UTM(11)` as long as it comes out as the same thing!

Reply:
Yes it does :) It took me a little while to find the right incantation. Ideally, we'd just accept `crs = landsat_5_img.rio.crs`.